system_upgrade.py

```diff
@@ -125,6 +126,20 @@ def checkDNFVer():
     if DNFVERSION < StrictVersion("1.1.0"):
         raise CliError(_("This plugin requires DNF 1.1.0 or later."))
 
+def enable_blanking(arg=10):
+    try:
+        tty = open('/dev/tty0', 'wb')
+    except Exception:
+        return
+    tty.write(b'\33[9;' + str(arg).encode('ascii') + b']')
+
+def maybe_disable_blanking():
+      term = os.getenv('TERM', '')
+      if not (term.startswith('linux') or term.startswith('con')):
+             return
```

Comment:
I had to comment out these two lines in order to make things work. With these two lines, screen still blanks, if I boot into plymouth splash and switch to text console afterwards.